docker-compose.override.yml

```diff
@@ -34,28 +39,35 @@ services:
     ]
     # Docker Compose does not set the TTY width, which causes Celery errors
     tty: false
-    env_file: ./dev/.env.docker-compose
+    environment:
+      UV_ENV_FILE: ./dev/.env.docker-compose
+    working_dir: /opt/django-project
     volumes:
       - .:/opt/django-project
+      - uv_cache:/var/cache/uv
     depends_on:
-      - postgres
-      - rabbitmq
-      - minio
+      postgres:
+        condition: service_healthy
+      rabbitmq:
+        condition: service_healthy
+      minio:
+        condition: service_healthy
 
   client:
     image: node:18
     working_dir: /app
     command: >
       sh -c "
         npm install &&
-        # If node_modules is not writable by others, make it so the host can modify and delete them
-        [ \"\$(stat -c '%A' node_modules | cut -c9)\" != 'w' ] && chmod o+w node_modules;
-        # Always run the dev server
```

Comment:
I thought I added this to the last review but maybe I forgot to click to add the comment.  This was done so that the host system when launched had access to delete and modify the node_modules folder without requiring a `sudo chown -R ./client/node_modules`.   If you want to delete the node_modules or `npm install` new dependencies the host machine won't have write access to the folder because inside the container `npm install` is run by `root`.

This is similar to how django manage.py migrate creates a migration file using the root user.  We can leave this for now.  I'll see if I can find a way to make this easier for both instances.

Reply:
yeah that means that it doesn't include a node_modules within the host.  It allows the npm install to work in terms of editing the package and package-lock but won't actually install the dependency locally.  I think it's fine for now and I'll review only if I start hitting annoying issues (which hopefulyl shouldn't happen often)